### PR TITLE
Skip multi az PD storage test if no extra zone detected

### PR DESCRIPTION
**Pull request by saikat-royc**
**What type of PR is this?**

**What this PR does / why we need it**:
If a cluster is deployed with nodes in all zones of a region, the test fails to find an extra zone and fatals.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by chendave — COMMENTED** [review]
This change looks good,  btw, the comments on this function seems like is out of the sync.
 > OnlyAllowNodeZones tests that GetAllCurrentZones returns only zones with Nodes

**Comment by saikat-royc** [comment]
> This change looks good, btw, the comments on this function seems like is out of the sync.
> 
> > OnlyAllowNodeZones tests that GetAllCurrentZones returns only zones with Nodes

Changed the comment on the function

**Comment by chendave** [comment]
/lgtm

thanks!

File: test/e2e/scheduling/ubernetes_lite_volumes.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ var _ = SIGDescribe("Multi-AZ Cluster Volumes [sig-storage]", func() {
 	})
 })
 
-// OnlyAllowNodeZones tests that GetAllCurrentZones returns only zones with Nodes
+// OnlyAllowNodeZones tests that PDs are only provisioned in zones with nodes.
```
Comment (alculquicondor) [inline test/e2e/scheduling/ubernetes_lite_volumes.go:64]: nit: s/nodes/Nodes

**Review by alculquicondor — COMMENTED** [review]
/approve

**Pull request merged**